### PR TITLE
feat: convert png, jpgs in WebP

### LINKED ISSUE #118: Serve images in next-gen formats
The images in the assets folder are in PNG or JPEG. Those images should be converted into next-gen formats like JPEG 2000, JPEG XR, or WebP for faster page loading time and improving the [PageSpeed score.](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fdeveloperfolio.js.org%2F&tab=mobile)

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #118

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
@naveen521kk review needed.

**Comment by aashutoshrathi** [comment]
@naveen521kk The recommended way is to keep both in secret as WebP is still not supported by many browsers.

**Comment by naveen521kk** [comment]
I am just asking to check out for other images as in the blogs.

File: convert_to_webp.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+for file in src/assets/images/*.{png,jpg}; do
```
Comment (naveen521kk) [inline convert_to_webp.sh:1]: Do we need this script? @saadpasta @kartik918

File: src/containers/contact/Contact.js [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
+                src={require("../../assets/images/contactMail.png")}
+              ></img>
+            </picture>
+          </div>
         </div>
-        </Fade>
-    );
+      </div>
+    </Fade>
+  );
```
Comment (kartik918) [inline src/containers/contact/Contact.js:52]: Avoid adding file formatting to your commits in the future. It makes it tough to review if there's a lot of files changed.

File: convert_to_webp.sh (hunk 1 above)
Comment (kartik918) [reply convert_to_webp.sh:1]: Seems like overkill and is compression something we want to "offer" to users. Making our images compressed seems like enough for the PR. Shouldn't add stuff that might confuse people.

Comment (naveen521kk) [reply convert_to_webp.sh:1]: @aashutoshrathi So can you remove the script.

Comment (aashutoshrathi) [reply convert_to_webp.sh:1]: Okay

**Comment by aashutoshrathi** [comment]
Thanks for that detailed comment, will push commits around that.

**Comment by naveen521kk** [comment]
@aashutoshrathi Any updates?

**Comment by aashutoshrathi** [comment]
@naveen521kk will update by EOD

**Comment by aashutoshrathi** [comment]
- Optimizations with WebP: https://github.com/saadpasta/developerFolio/pull/159/commits/f5ddaea93002847fb1e4c1d3a9384d6cbd9e81c2

File: src/containers/podcast/Podcast.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ export default function Podcast() {
         {podcastSection.podcast.map(podcastLink => {
           return (
             <div>
-              <iframe className= "podcast" src={podcastLink} frameborder="0" scrolling="no"></iframe>
+              <iframe title="Podcast" className= "podcast" src={podcastLink} frameborder="0" scrolling="no"></iframe>
```
Comment (naveen521kk) [inline src/containers/podcast/Podcast.js:18]: Changes in this file should be removed. This is not related with this PR.

**Comment by aashutoshrathi** [comment]
Moved to #191
